modules/oe_content_entity/tests/modules/oe_content_entity_test/oe_content_entity_corporate.links.action.yml

```diff
@@ -1,11 +1,11 @@
-entity.oe_corporate.add_page:
```

Comment:
Shouldn't this file name be the same as the module's? And do we need it at all as far as the tests we are running are concerned? If yes it can stay, after renaming.